src/uct/ib/rdmacm/rdmacm_listener.c

```diff
@@ -11,6 +11,25 @@
 #include "rdmacm_listener.h"
 
 
+#define UCS_RDMACM_MAX_BACKLOG_PATH        "/proc/sys/net/rdma_ucm/max_backlog"
+
+
+static long ucs_rdmacm_max_backlog()
+{
+    long max_backlog;
+
+    if (ucs_read_file_number(&max_backlog, 1,
+                             UCS_RDMACM_MAX_BACKLOG_PATH) == UCS_OK) {
+        ucs_assert(max_backlog <= INT_MAX);
+        return max_backlog;
+    } else {
+        ucs_warn("unable to read max_backlog value from %s file",
```

Comment:
is rdmacm able to work without this file?

Reply:
i don't know. but this warning will printed if this happens.

Reply:
let's make it ucs_diag